foundry/external_data_architectures.py

```diff
@@ -25,3 +26,23 @@ def __getitem__(self, idx):
         
         return item
     
+class FoundryDataset_TF(Sequence):
+    def __init__(self, inputs, targets):
+        self.inputs=inputs
+        self.targets=targets
+
+    def __len__(self):
+        return len(self.inputs[0])
+
+    def __getitem__(self, idx):
+        item = {"input": [], "target": []}
+        
+        for input in self.inputs:
```

Comment:
Should we care about batching, or can you use this class with a tensorflow dataloader chain?

Reply:
There should be ways that you can wrap this class inside another with batching. I don't think it's necessary for us to include batching functionality.